src/Template/Bake/Template/index.ctp

```diff
@@ -39,20 +39,21 @@ $fields = collection($fields)
     endforeach;
 %>
     </ul>
-</div>
-<div class="<%= $pluralVar %> index large-10 medium-9 columns">
+</nav>
+<div class="<%= $pluralVar %> index large-9 medium-8 columns" id="content">
```

Comment:
I am really missing point here. I think foundation zurb is a good choice.
yes having these classes is a common problem for separating presentation
and code but a person could easily disable the css and can use ids instead
at any time. so I added these ids on purpose. also I don't know ehat would
be the alternative or advantage of not using a css library. for me the
foundation was a huge improvement at using cake because I had routunes
from1.2 times changing standard ugly css to bootstrap and foundation. I
even wrote some js libraries to avoid these routines at that times.  the
problems talked here is another issue and common for css libraries. but the
frontend development is going somewhere different using schema.org or
structured data, machine readable information without any presentation. but
as a php fw for a base this is not our problem to solve. actually I would
love to create a new generator using only semantic elements without any
decoration and that would be a future improvement. I am open to all ideas
though.

11 Haziran 2015 Perşembe tarihinde, ADmad <notifications@github.com> yazdı:

> In src/Template/Bake/Template/index.ctp
> <https://github.com/cakephp/bake/pull/104#discussion_r32198806>:
>
> > @@ -39,20 +39,21 @@ $fields = collection($fields)
> >      endforeach;
> >  %>
> >      </ul>
> > -</div>
> > -<div class="<%= $pluralVar %> index large-10 medium-9 columns">
> > +</nav>
> > +<div class="<%= $pluralVar %> index large-9 medium-8 columns" id="content">
>
> @mtkocak <https://github.com/mtkocak> The benefit of basing the css off a
> standard css lib (zurb foundation in this case) is that people can easily
> add new elements as per their app needs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cakephp/bake/pull/104/files#r32198806>.
>